spec/spec_helper.rb

```diff
@@ -30,6 +30,11 @@
   end
   config.include FactoryBot::Syntax::Methods
 
+  config.before :all do
+    ActiveStorage::Current.url_options = { host: ENV.fetch('SERVER_HOST', nil),
+                                           port: ENV.fetch('PORT', 3000) }
+  end
```

Comment:
I don't think you need the before at all.. just 

```suggestion
  ActiveStorage::Current.url_options = { host: ENV.fetch('SERVER_HOST', nil),
                                         port: ENV.fetch('PORT', 3000) }
```

wdyt?

Reply:
also please, don't pollute the `spec_helper` with Rails related stuff. Move it either to `rails_helper` or maybe even to `config/environments/test.rb`